js/plugin/plugin.js

```diff
@@ -0,0 +1,10 @@
+jQuery( document ).ready(function() {
+	if (jQuery('input[value="wordpress-simplechart/simplechart.php"]').length) {
+		jQuery(
+			'input[value="wordpress-simplechart/simplechart.php"]'
+		)[0].disabled = true;
+		jQuery(
+			'input[value="wordpress-simplechart/simplechart.php"]'
+		).css('cursor', 'default');
+	}
```

Comment:
No need to repeat the jQuery selector here, and you can chain the jQuery methods to boot. Also, we try to follow [WP standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/) for JS in open source plugins.
```
var pluginInput = jQuery( 'input[value="wordpress-simplechart/simplechart.php"]' );
if ( pluginInput.length ) {
    jQuery( pluginInput ).attr( 'disabled', 'disabled' ).css( 'cursor', 'default' );
}
```
My jQuery is a little rusty, you might want to check that this works. Actually, it's so little code that you could just echo it in `admin_print_scripts` and wrap in `if ( 'plugins' === pagenow ) { ... }`